src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -37,6 +37,56 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>ControlsStrings.resx</DependentUpon>
     </Compile>
+    <Compile Update="Resources\ConsoleLogs.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ConsoleLogs.resx</DependentUpon>
+    </Compile>
```

Comment:
Would love to see a future PR that uses the ResX source generator for these files, rather than the custom tool. Less code in the repo, better features, no dependency on VS to update generated code.